wallet/src/de/schildbach/wallet/ui/invite/CreateInviteViewModel.kt

```diff
@@ -66,7 +66,7 @@ class CreateInviteViewModel @Inject constructor(
     private fun combineLatestData(): Boolean {
         val isSynced = blockchainStateData.value?.isSynced() ?: false
         val noIdentityCreatedOrInProgress = (blockchainIdentity.value == null) || blockchainIdentity.value!!.creationState == BlockchainIdentityData.CreationState.NONE
-        return isSynced && !noIdentityCreatedOrInProgress && canAffordIdentityCreation()
+        return isSynced && !noIdentityCreatedOrInProgress //&& canAffordIdentityCreation()
```

Comment:
No need for a `canAffordIdentityCreation` check anymore?

Reply:
We address the canAffordIdentityCreation in other places in the UI.

I will remove this commented code.